components/Card.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const Card = ({
+  action,
+  actionLabel,
+  address,
+  addressLabel,
+  backgroundImageSource,
+  children,
+  text,
+  title,
+}) => (
+  <div className="card shadow-xl image-full">
+    {backgroundImageSource && (
+      <figure>
+        <img src={backgroundImageSource} />
+      </figure>
+    )}
+    <div className="card-body justify-end">
+      {title && <span className="card-title">{title}</span>}
+      {text && <span className="mb-3">{text}</span>}
+      {action && (
+        <div className="pt-6">
+          <button className="btn btn-primary" onClick={action} type="button">
+            {actionLabel}
+          </button>
+        </div>
+      )}
+      {children}
```

Comment:
I'm not sure what your use case of `children` is expected to be when looking at other cards? 

Except maybe to allow additional paragraphs and text objects such as in `Health Education Workshop` where there are two paragraphs. In which case maybe it is better placed before the `action`.

Reply:
You're right. I forgot about several paragraphs. Will redisign it tomorrow... Somehow ;)

Reply:
Sounds like a plan, a case of balancing ease of modification and not doing my usual trick of over-engineering everything haha!